teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,58 +4,83 @@
    "cell_type": "markdown",
```

Comment:
<p>This function is supercool, but also quite complex, I'm wondering if it is understandable for beginners.</p><ul><li>Maybe if the reports are introduced in more detail above it becomes clear.</li><li>Also what could help might be a bit textual (markdown explanation) above the function?</li></ul><p>tbd</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>